src/main/java/application/onlinebookstore/service/OrderItemService.java

```diff
@@ -0,0 +1,13 @@
+package application.onlinebookstore.service;
+
+import application.onlinebookstore.dto.orderitem.CreateOrderItemDto;
+import application.onlinebookstore.dto.orderitem.OrderItemDto;
+import java.util.List;
+
+public interface OrderItemService {
+    OrderItemDto createOrderItem(CreateOrderItemDto orderItemDto);

```

Comment:
```suggestion
    OrderItemDto create(CreateOrderItemDto orderItemDto);

```
IMO OrderItem postfix can be omitted in OrderItemService, but it's for you to decide